pygmt/tests/test_clib_virtualfile_in.py

```diff
@@ -128,35 +128,3 @@ def test_virtualfile_in_matrix_string_dtype():
                 assert output == "347.5 348.5 -30.5 -30\n"
                 # Should check that lib.virtualfile_from_vectors is called once,
                 # not lib.virtualfile_from_matrix, but it's technically complicated.
-
-
-# TODO(PyGMT>=0.16.0): Remove this test in PyGMT v0.16.0 in which the old usage of
```

Comment:
There was a typo in #3722. This test should be removed in PyGMT v0.15.0, not PyGMT v0.16.0.